src/main/java/seedu/address/logic/commands/DeleteClientProfileCommand.java

```diff
@@ -25,6 +25,8 @@ public class DeleteClientProfileCommand extends Command {
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Successfully deleted %1$s "
             + "with the number: %2$s " + "and email: %3$s!";
+    public static final String MESSAGE_HAS_ACTIVE_LISTINGS = "Cannot delete client %1$s because they"
```

Comment:
For this error message, it may be more appropriate to say more specifically that the client is a `Seller` since we are not allowing `Sellers` with listings to be deleted from the `Listings`